requirements/test.txt

```diff
@@ -2,7 +2,7 @@
 -r base.txt
 Brotli==1.0.9
 coverage==5.3.1
-cryptography==3.2.1; platform_machine!="i686" and python_version<"3.9" # no 32-bit wheels; no python 3.9 wheels yet
+cryptography==3.3.1; platform_machine!="i686" and python_version<"3.9" # no 32-bit wheels; no python 3.9 wheels yet
```

Comment:
I don't think this is related to the PR.

Reply:
`pip-compile` is not able to pin dependencies without this change.